quantities/serialization/QuantityFactory.cs

```diff
@@ -0,0 +1,86 @@
+using System.Collections.Concurrent;
+using Quantities.Dimensions;
+using Quantities.Measures;
+
+namespace Quantities.Serialization;
+
+file static class Inject
+{
+    public static IInject Fractional { get; } = new FractionInjector();
+    public static IInject Multiplicative { get; } = new ProductInjector();
```

Comment:
The multiplicative injector is for quantities like `km/h`?

Reply:
It's for Energy, i.e. "Watt Hour" (Wh).

Contrary to "fractional" quantities such as velocity (`km/h`) the multiplicative symbol "*" is not used, as the "multiplication" is implied.

Reply:
Thanks 😊